testing/e2e/app/routes/InformationPanel/spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect, type Page } from '@playwright/test';
+
+test.describe('Information Panel Resizing', () => {
+  test('should adjust information panel width', async ({ page }) => {
+    await page.goto('/InformationPanel', { waitUntil: 'domcontentloaded' });
+    const infoPanel = page.locator('#InformationPanel');
+    const initialWidth = (await infoPanel.boundingBox())?.width;
+    expect(await infoPanel.getAttribute('style')).toBeFalsy();
+
+    await resizeInformationPanel(page);
+
+    const newWidth = (await infoPanel.boundingBox())?.width;
+    expect(newWidth).toBe((initialWidth as number) - 100);
+    expect(await infoPanel.getAttribute('style')).toBeTruthy();
+  });
+  test('should reset styles on orientation change', async ({ page }) => {
+    await page.goto('/InformationPanel', { waitUntil: 'domcontentloaded' });
+    const infoPanel = page.locator('#InformationPanel');
+    expect(await infoPanel.getAttribute('style')).toBeFalsy();
+
+    await resizeInformationPanel(page);
+
+    expect(await infoPanel.getAttribute('style')).toBeTruthy();
```

Comment:
Instead of checking for the `style` attribute, we should be checking that the dimensions actually changed.

When changing orientation from vertical to horizontal, the panel width should reset to be the same width as the wrapper. And when changing the orientation back to vertical, the height should reset to be the same as the wrapper.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/d272dbf14dd66a774a396cf860087905f17863d2)